content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -28,13 +28,22 @@ in this struct. Make sure the Kubelet has read permissions on the file.
 
 Here is an example of what this file might look like:
 ```
-apiVersion: kubelet.config.k8s.io/v1beta1
-kind: KubeletConfiguration
-address: "192.168.0.8"
-port: 20250
-serializeImagePulls: false
-evictionHard:
-    memory.available:  "200Mi"
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubelet-config-1.20
+  namespace: kube-system
+data:
+  kubelet: |
+    apiVersion: kubelet.config.k8s.io/v1beta1
+    kind: KubeletConfiguration
+    address: "192.168.0.8"
+    port: 20250
+    serializeImagePulls: false
+    evictionHard:
+      memory.available:  "200Mi"
+~                                  
```

Comment:
What is this line?

Reply:
@tengqm I've resubmitted a PR since I realised there were errors in the one he submitted and I mistakenly allowed it to merge.

Reply:
Those lines were in original config examples too.
https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/

Reply:
@bhangra The '~' and the empty line was introduced by this PR which was merged and published.